thoth/python/aiosource.py

```diff
@@ -268,3 +268,44 @@ async def get_package_versions(self, package_name: str) -> AsyncIterableVersions
 
         package_info = await self._warehouse_get_api_package_info(package_name)
         return AsyncIterableVersions(list(package_info["releases"].keys()))
+
+    async def get_package_artifacts(self, package_name: str, package_version: str):
+        """Return list of artifacts corresponding to package name and package version."""
+        to_return = []
+        possible_continuations = [".win32", ".tar.gz", ".whl", ".zip", ".exe", ".egg", "-"]
+        async for artifact_name, artifact_url in await self._simple_repository_list_artifacts(package_name):
+            # Convert all artifact names to lowercase - as a shortcut we simply convert everything to lowercase.
+            artifact_name.lower()
+
+            for i in possible_continuations:
+                if artifact_name.startswith(f"{package_name}-{package_version}{i}"):
```

Comment:
Does this work also with platform tags in wheel distributions?

https://www.python.org/dev/peps/pep-0425/

Reply:
Yes because `-` is included in possible continuations

Reply:
Will it match for example `tensorflow-2.2.0-cp36-cp36m-manylinux2010_x86_64.whl`?

https://pypi.org/project/tensorflow/#files

Reply:
Yeah, that starts with `tensorflow-2.2.0-`

Reply:
Ah, I didn't see the `-` trick :)